pkg/collector/corechecks/cluster/kubernetes_state.go

```diff
@@ -32,7 +33,8 @@ import (
 
 const (
 	kubeStateMetricsCheckName = "kubernetes_state_core"
-	defaultResyncPeriod       = 30
+	defaultResyncPeriod       = 30 // in seconds
```

Comment:
I think 30sec is very short for resync

Reply:
good point, we could use `kubernetes_informers_resync_period` which is 5 minutes by default